cfg4k-core/src/test/kotlin/com/jdiazcano/cfg4k/TimedReloadStrategyTest.kt

```diff
@@ -78,8 +81,47 @@ nested.a=reloaded nestedb
             )
             checkProvider(overrideFile, provider, text, true)
         }
-    }
 
+        it("error on reload doesn't prevent further reload attempts") {
```

Comment:
I like this test!

Reply:
Well, now that the contract of `ConfigProvider` is to not to throw any errors, and the try-catch above is a surplus, this test is becomes wrong. So for this PR I'm going to remove it.
However, it might be reused to test catching of errors in the `DefaultConfigProvider.reload()` you mentioned above.

Reply:
I think it is good to leave it there, it proves that we are catching the exception when something happens. I think this was a miss on my side when added the reload error listener.